presto-product-tests/bin/run_on_docker.sh

```diff
@@ -150,7 +150,10 @@ docker version
 
 for available_docker_environment in $(getAvailableEnvironments)
 do
-    stop_docker_compose_containers "${PRODUCT_TESTS_ROOT}/conf/docker/${available_docker_environment}/docker-compose.yml"
+    if [[ -e "${PRODUCT_TESTS_ROOT}/conf/docker/${available_docker_environment}/docker-compose.yml" ]]
```

Comment:
That file does not exist in the "common" directory, so having it error out means the script will never run.  I'll look for a better work-around for that issue.